browser/extensions/api/settings_private/brave_prefs_util.cc

```diff
@@ -299,8 +299,6 @@ const PrefsUtil::TypedPrefMap& BravePrefsUtil::GetAllowlistedKeys() {
 #if BUILDFLAG(ENABLE_TOR)
   (*s_brave_allowlist)[tor::prefs::kAutoOnionRedirect] =
```

Comment:
I believe PR should dance around "auto redirect onion location" instead of "onion only in tor" or I misunderstood something?

Reply:
Thank you, I swapped the two toggles/options in my head - not even sure why!